### PR TITLE
Feature/add delete all nodes

### PR DESCRIPTION
**Pull request by mr-rooftop**

### CODE REVIEW

**Review by mbartn — APPROVED** [review]
veri nice

File: bot/helpers.py [hunk 1]
```diff
@@ -128,9 +140,20 @@ def try_message(context, chat_id, text, reply_markup=None):
...
+def add_node_to_user_data(user_data, address, node):
     """
-    Return json of desired validator node
+    Add a node in the user specific dictionary
+    """
+
+    user_data['nodes'][address] = {}
+    user_data['nodes'][address]['status'] = node['status']
+    user_data['nodes'][address]['jailed'] = node['jailed']
+    user_data['nodes'][address]['delegator_shares'] = node['delegator_shares']
```
[the remark this answers is not in the document]
Comment (mr-rooftop) [reply bot/helpers.py:151]: Well, we don't need all the fields, do we?

File: bot/helpers.py [hunk 2]
```diff
@@ -140,8 +163,29 @@ def get_validator(address):
             logger.info("ConnectionError while requesting " + VALIDATORS_ENDPOINT)
             raise ConnectionError
         nodes = response.json()
+        return nodes['result']
+    else:
+        response = requests.get(VALIDATORS_ENDPOINT)
+        if response.status_code != 200:
+            if not is_lcd_reachable:
```
[the remark this answers is not in the document]
Comment (mr-rooftop) [reply bot/helpers.py:170]: Oh shit..

File: bot/terra_node_bot.py [hunk 3]
```diff
@@ -246,15 +253,79 @@ def handle_add_node(update, context):
...
+    """
+
+    query = update.callback_query
+
+    addresses = []
+    for address in context.user_data['nodes']:
+        addresses.append(address)
+
+    for address in addresses:
+        del context.user_data['nodes'][address]
```
[the remark this answers is not in the document]
Comment (mr-rooftop) [reply bot/terra_node_bot.py:320]: omg it's the king of PR reviews

**Pull request merged**